### PR TITLE
Update to `context.tf`.

### PR DESCRIPTION
**Pull request by htplbc**
## what
* Update to `context.tf`

## why
* Keep the module up to date

### CODE REVIEW

**Review by danjbh — DISMISSED** [review]
LGTM

File: context.tf [hunk 1]
```diff
@@ -19,7 +19,8 @@
 #
 
 module "this" {
-  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source  = "cloudposse/label/null"
+  version = "0.22.0"
```
Comment (aknysh) [inline context.tf:23]:
```suggestion
  version = "0.22.1"
```

File: examples/complete/context.tf [hunk 2]
```diff
@@ -19,7 +19,8 @@
 #
 
 module "this" {
-  source = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.19.2"
+  source  = "cloudposse/label/null"
+  version = "0.22.0"
```
Comment (aknysh) [inline examples/complete/context.tf:23]:
```suggestion
  version = "0.22.1"
```

**Review by aknysh — COMMENTED** [review]
please run these commands:

```
make init
make github/init
make readme
```

**Review by aknysh — APPROVED** [review]

**Pull request merged**